app/models/govuk_taxonomy/branches.rb

```diff
@@ -0,0 +1,50 @@
+module GovukTaxonomy
+  class Branches
+    HOMEPAGE_CONTENT_ID = "f3bbdec2-0e62-4520-a7fd-6ffd5d36e03a".freeze
+
+    def branch_name_for_content_id(content_id)
+      get_content_item(content_id).dig('title')
+    end
+
+    def all
```

Comment:
I've realised why it's this way - the `transform` method needs to know if each taxon is published or draft.